src/main/java/com/softserveinc/ita/pageobjects/BasePage.java

```diff
@@ -22,14 +27,38 @@ public CatalogModal openCatalog() {
         return new CatalogModal();
     }
 
-    public T searchBarInputText(String inputText) {
-        $x("//input[@name = 'search']").setValue(inputText);
+    public T searchBarInputField(String inputText) {
```

Comment:
setTextInSearchBar or something like this

Reply:
done